geninfo/genisolist.ini

```diff
@@ -66,37 +66,51 @@ platform = $4
 key_by = $2 $4
 sort_by = $4 $2
 
-[ubuntu desktop]
+[ubuntu_desktop]
 distro = Ubuntu
 listvers = 4
-# If one glob is not enough for locating all images, you can use "location_N"
-# to specify more globs. N start from 0 and must a sequence (0,1,2...).
-location_0 = ubuntu-releases/[a-z]*/ubuntu-*-desktop-i386.iso
-location_1 = ubuntu-releases/[a-z]*/ubuntu-*-desktop-amd64.iso
+location = ubuntu-releases/[a-z]*/ubuntu-*-desktop-*.iso
 pattern = ubuntu-([0-9.]+)-desktop-(\w+).iso
 version = $1
 type = Desktop LiveDVD
 platform = $2
 
-[ubuntu live server]
+[ubuntu_preinstalled_desktop]
+distro = Ubuntu
+listvers = 4
+location = ubuntu-cdimage/releases/[a-z]*/release/ubuntu-*-preinstalled-desktop-*.img.xz
```

Comment:
This kind of entries are not generated.

I can find file that matches the location
```
./releases/kinetic/release/ubuntu-22.10-preinstalled-desktop-arm64+raspi.img.xz
./releases/jammy/release/ubuntu-22.04.1-preinstalled-desktop-arm64+raspi.img.xz
```